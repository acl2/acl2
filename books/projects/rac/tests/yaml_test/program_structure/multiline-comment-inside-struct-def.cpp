// RAC begin

struct S {

  int a; /* A
            long
            comment
        */
};

int foo() {
} // Error here (we check the location)
