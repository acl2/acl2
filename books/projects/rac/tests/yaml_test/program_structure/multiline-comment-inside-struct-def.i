# 0 "<stdin>"
# 0 "<built-in>"
# 0 "<command-line>"
# 1 "/usr/include/stdc-predef.h" 1 3 4

# 1 "/usr/include/stdc-predef.h" 3 4
/* Copyright (C) 1991-2024 Free Software Foundation, Inc.
   This file is part of the GNU C Library.

   The GNU C Library is free software; you can redistribute it and/or
   modify it under the terms of the GNU Lesser General Public
   License as published by the Free Software Foundation; either
   version 2.1 of the License, or (at your option) any later version.

   The GNU C Library is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
   Lesser General Public License for more details.

   You should have received a copy of the GNU Lesser General Public
   License along with the GNU C Library; if not, see
   <https://www.gnu.org/licenses/>.  */




/* This header is separate from features.h so that the compiler can
   include it implicitly at the start of every compilation.  It must
   not itself include <features.h> or any other header that includes
   <features.h> because the implicit include comes before any feature
   test macros that may be defined in a source file before it first
   explicitly includes a system header.  GCC knows the name of this
   header in order to preinclude it.  */

/* glibc's intent is to support the IEC 559 math functionality, real
   and complex.  If the GCC (4.9 and later) predefined macros
   specifying compiler intent are available, use them to determine
   whether the overall intent is to support these features; otherwise,
   presume an older compiler has intent to support these features and
   define these macros by default.  */
# 56 "/usr/include/stdc-predef.h" 3 4
/* wchar_t uses Unicode 10.0.0.  Version 10.0 of the Unicode Standard is
   synchronized with ISO/IEC 10646:2017, fifth edition, plus
   the following additions from Amendment 1 to the fifth edition:
   - 56 emoji characters
   - 285 hentaigana
   - 3 additional Zanabazar Square characters */
# 0 "<command-line>" 2
# 1 "<stdin>"
# 0 "tests/yaml_test/program_structure/multiline-comment-inside-struct-def.cpp"

# 0 "tests/yaml_test/program_structure/multiline-comment-inside-struct-def.cpp"
// RAC begin

struct S {

  int a; /* A
            long
            comment
        */
};

int foo() {
} // Error here (we check the location)
