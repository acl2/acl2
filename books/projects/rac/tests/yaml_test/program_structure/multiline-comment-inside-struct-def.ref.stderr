multiline-comment-inside-struct-def.cpp:12 (0-1): syntax error, unexpected '}'
 12 | } // Error here (we check the location)
    | ^
